.github/workflows/pull_request.yml

```diff
@@ -99,6 +99,54 @@ jobs:
           PY_COLORS: '1'
           ANSIBLE_FORCE_COLOR: '1'
         shell: bash
+  core:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.1
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Install prerequisites
+        run: |
+          pip install --upgrade pip
+          sudo apt remove ansible
+          pip install --pre ansible-core
+          pip install molecule docker molecule-docker ansible-lint[yamllint]
+          ansible-galaxy collection install --force community.general -p ./temp
+          ansible-galaxy collection install --force community.crypto -p ./temp
+          mkdir -p plugins/modules
+          cp temp/ansible_collections/community/general/plugins/modules/files/ini_file.py plugins/modules
+          cp temp/ansible_collections/community/general/plugins/modules/system/seport.py plugins/modules
+          cp temp/ansible_collections/community/crypto/plugins/modules/openssl_privatekey.py plugins/modules
+          cp temp/ansible_collections/community/crypto/plugins/modules/openssl_publickey.py plugins/modules
+
+          cp temp/ansible_collections/community/general/plugins/modules/system/make.py plugins/modules
+          cp temp/ansible_collections/community/general/plugins/modules/packaging/language/pip_package_info.py plugins/modules
+
```

Comment:
@Spredzy this is interesting what they are doing here. Would we want to consider the same, vendoring the code at build time instead of committing it to the repo?